code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -484,6 +497,10 @@ func (uar *updateAllocationRequest) validate(
 		}
 	}
 
+	if uar.FileOptions > 63 || uar.FileOptions < 0 {
```

Comment:
uar.FileOptions being uint8 can never be < 0. we can remove this check

Reply:
Right, will remove!

Reply:
Donezo!! :)